infra-templates/k8s/30/README.md

```diff
@@ -0,0 +1,25 @@
+## Kubernetes 1.7.0
+
+### Upgrading to this Version
```

Comment:
Can you update the read me to remove the upgrading section. THis will only be for Fresh installs.